p4-16/spec/P4-16-spec.mdk

```diff
@@ -1111,11 +1111,7 @@ excerpts from the grammar:
 - `UPPERCASE` symbols denote terminals in the grammar.
 - Excerpts from the grammar are given in BNF notation as follows:
 ~ Begin P4Grammar
-p4program
-    : /* empty */
-    | p4program declaration
-    | p4program ';'
-    ;
+[INCLUDE=grammar.mdk:p4program]
```

Comment:
this looks great.

Reply:
I assume that madoko will complain if the link is broken

Reply:
I just ran a local test on my system (not part of this PR) where I changed the INCLUDE line above to have a name that was not defined in the grammar.mdk file: `[INCLUDE=grammar.mdk:p4progra]` (I deleted the final character of `p4program` to introduce a mistake.  Madoko produces a warning near the beginning of its output, as shown below on the Ubuntu 20.04 system where I tested this:
```
$ make
madoko --pdf -vv --png --odir=build P4-16-spec.mdk
process: P4-16-spec.mdk -> build/P4-16-spec.html
  warning: P4-16-spec.mdk:1114: unknown snippet name: p4progra

```